### DISCUSSION TITLE
Solace Pubsub+ event broker auto scaling ideas

### DISCUSSION THREAD

**Discussion by dirkckau** [kedacore/keda#3412] [Ideas]
Hi all, 

Shall we add the "unsafeSsl" feature for the Solace Pubsub+ event broker auto scaling? 
The reason behind is that the management protocol of Solace "SEMP" is now supporting SSL. 
It will be nice if we could enable the "unsafeSsl" feature for the Solace scaled object, which is now only supporting the "solaceSempBaseURL". 

Another idea is that the "solaceSempBaseURL" is now only supporting one endpoint, if we can support 2 endpoint? 
The reason behind is that Solace is running in Active/Standby mode, if we could allow 2 endpoints monitoring the same vpn and queue, and using the higher one, it would fit most enterprise resilience case. 

Regards,
Dirk

**Comment by zroubalik** [comment]
Hi, could you please open a separate issues for those tasks? As for `unsafeSsl` I think that we it could be added for the latter ask, I think that we should discuss that with some Solace experts.

Are you willing to contribute this? :)
